thunder/tests/framework.py

```diff
@@ -62,6 +62,19 @@ class NOTHING:
 IS_WINDOWS = platform.system() == "Windows"
 
 
+def _bitsandbytes_available():
+    if not package_available("bitsandbytes"):
+        return False
+    try:
+        import bitsandbytes
+    except Exception:
```

Comment:
Thank you @kiya00. Would we want to make this more specific, eg `(ImportError, RuntimeError)` like below?

Reply:
Thank you @t-vi , I've added it